### DISCUSSION TITLE
Is there any way to do partial update?

### DISCUSSION THREAD

**Discussion by prateekgoel** [vitusortner/floor#589] [General]
Hi,
I need to update the data in database partially.
I am receiving updated fields of an entity from backend.
If i call update on entity using that data, rest of the fields will become null, so I cannot do that.

I do no want to write a @Query for update bcz the number of fields are more and are dynamic (comes from server).

So, is there any way I can update the data partially? (update only the columns that i pass in arguments and keep the rest to the existing values)

 Android Room does have that feature, but I couldn't find anything in floor.
 
 @vitusortner @mqus

**Comment by mqus** [comment]
No, currently there isn't.

**Reply by prateekgoel** [reply]
We should have this feature then.
